windows/ReactNativeWebView/ReactWebView2.cpp

```diff
@@ -121,14 +121,12 @@ namespace winrt::ReactNativeWebView::implementation {
             });
 
         if (m_messagingEnabled) {
-            winrt::hstring windowAlert = L"window.alert = function (msg) {window.chrome.webview.postMessage(`{\"type\":\"__alert\",\"message\":\"${msg}\"}`)};";
-            winrt::hstring postMessage = L"window.ReactNativeWebView = {postMessage: function (data) {window.chrome.webview.postMessage(String(data))}};";
-            webView.ExecuteScriptAsync(windowAlert);
-            webView.ExecuteScriptAsync(postMessage);
+            winrt::hstring message = LR"(window.alert = function (msg) {window.chrome.webview.postMessage(`{"type":"__alert","message":"${msg}"}`)}; window.ReactNativeWebView = {postMessage: function (data) {window.chrome.webview.postMessage(String(data))}};)";
```

Comment:
Nit, break this up in a couple of lines within the same string, for example:

```suggestion
           winrt::hstring message = LR"(
               window.alert = function (msg) {window.chrome.webview.postMessage(`{"type":"__alert","message":"${msg}"}`)};
               window.ReactNativeWebView = {postMessage: function (data) {window.chrome.webview.postMessage(String(data))}};)";
```